src/fsc/fsc.targets

```diff
@@ -44,12 +44,16 @@
 
   <ItemGroup>
     <PackageReference Include="System.Console" Version="$(SystemConsoleVersion)" />
+    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="$(SystemDiagnosticsDiagnosticSourceVersion)" />
     <PackageReference Include="System.Linq.Expressions" Version="$(SystemLinqExpressionsVersion)" />
     <PackageReference Include="System.Memory" Version="$(SystemMemoryVersion)" />
     <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="$(SystemRuntimeCompilerServicesUnsafeVersion)" />
     <PackageReference Include="System.Reflection.Metadata" Version="$(SystemReflectionMetadataVersion)" />
     <PackageReference Include="System.Runtime.Loader" Version="$(SystemRuntimeLoaderVersion)" />
     <PackageReference Include="System.Security.Principal" Version="$(SystemSecurityPrincipalVersion)" />
+    <PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.1.0" />
+    <PackageReference Include="OpenTelemetry.Exporter.Zipkin" Version="1.1.0" />
```

Comment:
I think the most defensible addition here is the bare protocol - if end users want to consume this then they'll have to set up and configure the [collector](https://opentelemetry.io/docs/collector/). That keeps us endpoint-agnostic and only binding to the exporter protocol.

Reply:
Actually I'm going to remove this.
~This PR intends to only add activities in the service/background checker, not the compiler exe~.
This PR doesn't setup any exporters in the compiler exe the way your initial changes did - see `fscmain.fs`.

So the setup of the exporters happens in user code (eg. IDE, or external test code).

Have reverted these two lines as well.

Reply:
Long-term I think it would be best to allow users to specify an exporter via "late bindings" - as nasty as that sounds, I think it's better than asking users to setup one more backend service to forward the data.

But I hope this PR doesn't require making that decision.